dev/integration_tests/web_e2e_tests/README.md

```diff
@@ -0,0 +1,7 @@
+# Flutter for Web Integration testing with `integration_test` package
+
+These are web tests which are using `integration_test` (aka e2e) package. In order to run them locally, run `chromedriver` on port 4444. Later use the following command:
```

Comment:
Can you include the command for running `chromedriver` on port 4444?

Reply:
done. I also added links for more information.